bsp/esp-box/idf_component.yml

```diff
@@ -19,7 +19,7 @@ dependencies:
     public: true
 
   button:
-    version: "^2.5"
```

Comment:
Minimal version for >=IDF5 is 2.4.1 (there is new ADC driver)
- custom buttons are available from 2.5 (main button)